aspnetcore/performance/ObjectPool.md

```diff
@@ -50,7 +48,14 @@ The ObjectPool can be used in an app in multiple ways:
 
 ## How to use ObjectPool
 
-Call <xref:Microsoft.Extensions.ObjectPool.ObjectPool`1> to get an object and <xref:Microsoft.Extensions.ObjectPool.ObjectPool`1.Return*> to return the object.  There's no requirement that you return every object. If you don't return an object, it will be garbage collected.
+Call <xref:Microsoft.Extensions.ObjectPool.ObjectPool`1.Get*> to get an object and <xref:Microsoft.Extensions.ObjectPool.ObjectPool`1.Return*> to return the object.  There's no requirement that you return every object. If you don't return an object, it will be garbage collected.
```

Comment:
Just specified `Get` as method here. Before it pointed to the pool-class.

Reply:
```
::: moniker range=">= aspnetcore-3.0"

// 3.0 and higher
::: moniker-end

::: moniker range="< aspnetcore-3.0"


::: moniker-end
```